pkg/minikube/bootstrapper/kubeadm/kubeadm.go

```diff
@@ -75,7 +75,7 @@ func NewBootstrapper(api libmachine.API) (*Bootstrapper, error) {
 	if err != nil {
 		return nil, errors.Wrap(err, "command runner")
 	}
-	return &Bootstrapper{c: runner, contextName: name, k8sClient: nil}, nil
+	return &Bootstrapper{c: runner, contextName: viper.GetString(config.MachineProfile), k8sClient: nil}, nil
```

Comment:
contextName feels like it shouldn't map to a machine profile - whatever that is.

Reply:
our kubectl context name has always mapped one to one with the name of the cluster. constants.MachineProfile, bad as the name is, is just the name of profile passed into whatever minikube command is run. that's the reason I'm using that directly.

Reply:
OK. Seems reasonable - kubeadm bootstrapper uses contextName to find the apiserver to talk to.